discord.md

```diff
@@ -0,0 +1,30 @@
+/*
+Title: Discord Integration 
+Sort: 3
+*/
+
+Thats right Nightbot is now usable on your very own Discord Server! 
+
+No idea what Discord is? Discord is an all-in-one voice and text chat for gamers that’s free, secure, and works on both your desktop and phone. Check them out at [discordapp.com](https://discordapp.com/)!
+
+## Set-up
+
+1. In order to have Nightbot in your Discord server, you need a Discord Server! If you haven't already, head over to [discordapp.com](https://discordapp.com/) and make yourself a profile and create a server. More info on everything Discord can be found on their [help pages](https://support.discordapp.com/hc/en-us).
+
+2. Log in to Nightbot and head to the [Integrations](https://beta.nightbot.tv/integrations) tab. Here integrations settings for Discord can be found and managed.
+
+3. Clicking Connect will open a pop-out where you can allow Nightbot to access your Discord. (Don't worry we just need your Username and Avatar) Upon granting access you can add a server in which for Nightbot to join. Select a server from the dropdown list and keep the permissions the same. **Note:** Nightbot will **not** kick or ban users from your server.
+
+4. Nightbot should now be in your Discord Server! He has also created a new server role named "Nightbot". (try giving him a color) All of your Commands will be functional in your Discord, so try them out. 
+
+    The next steps are optional but are useful to sync Moderators, Regulars and Subscribers to your Discord and Spam Filtering.
```

Comment:
selecting roles is required if because of command userlevels.. nightbot needs to know each user's permission